apps/web/src/app/[locale]/(dashboard)/[wsId]/inventory/products/[productId]/form.tsx

```
@@ -132,206 +151,162 @@
+                                  )}
+                                >
+                                  <SelectValue
+                                    placeholder={t(
+                                      'ws-inventory-warehouses.placeholder'
+                                    )}
+                                  />
+                                </SelectTrigger>
+                              </FormControl>
+                              <SelectContent>
+                                {warehouses.map((warehouse) => (
+                                  <SelectItem value={warehouse.id}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)